src/solvers/dg_tree/dg_3d.jl

```diff
@@ -1033,7 +1034,8 @@ function calc_mortar_flux!(surface_flux_values,
     for j in eachnode(dg), i in eachnode(dg), v in eachvariable(equations)
       fstar_lower_right[v, i, j] += noncons_diamond_lower_right[v, i, j]
     end
-    mortar_fluxes_to_elements!(surface_flux_values, equations, mortar_l2, dg, cache, mortar,
+    mortar_fluxes_to_elements!(surface_flux_values, mortar_l2,
+                               mesh, equations, dg, cache, mortar,
```

Comment:
Argument ordering

Reply:
Why is this even working? Isn't the function definition itself in a different order?

Reply:
Yes, that's why it's not working.